draft-packages/avatar/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "extends": "../../tsconfig.dist.json",
+  "compilerOptions": {
+    "composite": true,
+    "allowJs": false,
+    "declaration": true,
+    "declarationDir": "dist",
+    "noEmit": false,
+    "sourceMap": true,
+    "outDir": "dist",
+    "rootDir": "src",
+    "tsBuildInfoFile": "./dist/.cache/tsbuildinfo.json"
+  },
+  "include": ["src", "styles"],
+  "exclude": ["**/*.stories.*"],
+  "references": [
+    {
+      "path": "../../packages/component-library"
```

Comment:
Should peer dependencies (kaizen/design-tokens) be included as a reference?

Reply:
OH! Yes they should - this slipped my mind as _most_ packages don't consume the types (rather the SCSS variables) from those packages. Will fix.